src/daemons/balanceDaemon.ts

```diff
@@ -15,9 +16,9 @@ runDaemon(async tools => {
   for await (const eventRow of streamEvents(connection, 'address-balance', {
     afterDate: safeDate(Date.now() - msBack)
   })) {
-    await checkEventTrigger(tools, eventRow).catch(error => {
```

Comment:
Don't edge conventions say to use "error" for this? If it's not in the conventions, I know we had a sidebar about this a few years ago, since I've been doing it ever since then.

Reply:
Pino's API expects `msg` and `err`. I can keep the parameter name as `error` if you'd like.